src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -37,4 +39,15 @@ public static int countEven(int[] array) {
         }
         return evenElements;
     }
+
+    public static int[] copy(int[] array){
+        if(Objects.isNull(array)){
```

Comment:
For consistency, please use `com.google.common.base.Preconditions.checkNotNull` method as it has been already used instead of `Objects.isNull`